[nano-signal-slot] Update to latest version

RT2Code opened the pull request:
I know that official releases should be preferred, but I used the latest commit instead because it fix some compilation errors on the latest MSVC version (https://github.com/NoAvailableAlias/nano-signal-slot/issues/26).

RT2Code committed:
[nano-signal-slot] Update to latest version

I know that official releases should be preferred, but I used the latest commit instead because it fix an issue which was causing compilations error on the latest MSVC version (https://github.com/NoAvailableAlias/nano-signal-slot/issues/26).

NancyLi1013 commented on ports/nano-signal-slot/CONTROL:
```diff
@@ -1,3 +1,4 @@
 Source: nano-signal-slot
-Version: 2018-08-25-1
+Version: 20-12-2019
```
The Version field seems some problem. 
Could you update it as 2019-12-20?

RT222 replied:
My bad, that's the French date format, I changed it.

RT2Code committed:
[nano-signal-slot] Fix wrong date format

RT2Code committed:
[nano-signal-slot] Version update

The developer of this library has just added a new release version including the MSVC compilation bug fix.

ras0219-msft commented:
Thanks for the PR!
